joomla/joomla-cms
# [com_tags] Fix field name typo

### Summary of Changes
Related to PR #23593.

There is a typo in `all_tags_show_tag_descripion`, thus , this setting is always enabled on the frontend.

### Testing Instructions
Code review
or
Install demo
Go `Global Configuration > Tags > List All Tags`
Toggle `Tag Descriptions` and check frontend under `All Tags`.
